CHANGELOG.md

```diff
@@ -5,9 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 1.2.0 - 2020-04-2x
+### Added
+- Official support for Kubernetes
+- Official support for Windows (Windows Server 2008,Windows Server 2012, Windows Server 2016, Windows Server 2019)
```

Comment:
we don't support Server 2008 on the Infra agent; this should be aligned with our official support: https://docs.newrelic.com/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure#operating-systems

Reply:
nice catch. I just copy pasted from some other place.